crates/typst-library/src/text/raw.rs

```diff
@@ -30,6 +30,12 @@ use crate::World;
 /// Displays the text verbatim and in a monospace font. This is typically used
 /// to embed computer code into your document.
 ///
+/// Note that text given to this element cannot contain arbitrary formatting,
+/// such as `{*strong*}` or `{_emphasis_}`, as it is displayed verbatim. If
```

Comment:
```suggestion
/// such as `[*strong*]` or `[_emphasis_]`, as it is displayed verbatim. If
```

I think this is what you meant.

Reply:
No, \`{...}\` is used elsewhere to syntax highlight expressions.

However, I'm now seeing that the expressions are all in code mode already, so this could fail to provide the desired highlighting. Perhaps `{[*strong*]}` would be needed to highlight properly, but i have a feeling that this could detract from the original purpose of the phrase. An alternative is ``[`{strong}`]($strong)`` and ``[`{emph}`]($emph)``, albeit less self-evident.

Reply:
`{...}` gives code highlighting and `[...]` gives markup highlighting. The brackets are stripped in both cases.

Reply:
Oh, I thought that brackets could only be interpreted differently for links. This should indeed be adjusted that way then.